plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/VidarrPlugin.java

```diff
@@ -141,6 +189,22 @@ public UnloadWorkflowRunsAction unloadByWorkflowRuns() {
     return new UnloadWorkflowRunsAction(definer);
   }
 
+  private String getMaxInFlightMessage(Optional<URI> url, String workflow) {
+    String message;
+    mifCache.setURL(url);
+    Optional<MaxInFlightDeclaration> maxInFlight = mifCache.get();
+    if (maxInFlight.isPresent()) {
+      message =
+          String.format(
+              "%d of max %d",
+              maxInFlight.get().getWorkflows().get(workflow).getCurrentInFlight(),
```

Comment:
Now that I think about it, `.get(workflow)` might return null.

Reply:
Thanks, fixed. Given that this won't build without the updates to Vidarr, should I wait before merging?

Reply:
Yes. Please wait until Vidarr is released. I think Angie is release lead this week, so if you want to point her to this PR, she can merge it when doing the release.

Reply:
OK, will do.